google_compute_engine/constants.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A module for global constants."""
+
+import platform
+
+if platform.system() == 'FreeBSD':
+    LOCALBASE = '/usr/local'
```

Comment:
Overall this change looks reasonable. One last request - would you mind separating the words in the variable names with underscores to make it easier to read?